portfolio/src/main/webapp/comment.html

```diff
@@ -22,7 +22,17 @@
           <button type="submit">Submit</button>
         </form>
       </section>
-      <section id="comment-display" class="comment-feed-container"></section>
+      <section class="comment-feed-container">
+        <form class="container radio" method="POST">
```

Comment:
I don't think this form is actually working as intended, it needs an `action`, which could be a javascript function or a URL. But it doesn't matter, as the form is never submitted, so probably the `method="POST"` is not needed either.

https://developer.mozilla.org/en-US/docs/Learn/Forms

Reply:
I incorrectly assumed radio buttons could only be used within a form--as I just want the radio button functionality and never want to actually submit anything, I changed the container to be an article instead (as they can stand alone).